features/delete_system_from_conf.feature

```diff
@@ -1,15 +0,0 @@
-# Copyright (c) 2015 SUSE LLC
-# Licensed under the terms of the MIT license.
-
-Feature: Remove system from configuration channel
-
-  Scenario: Remove system from configuration channel
```

Comment:
Why not using this scenario as cleanup scenario for `Scenario: I add a config file deployment to the action chain` + some steps to remove the deployed files on the "sle-client" ?